scripts/lake_sheltering_nlcd.R

```diff
@@ -1,4 +1,45 @@
 
+
+get_nlcd_classes <- function(lake.buffer, filename){
```

Comment:
I added a different function to get the full classes for the work we did for the MGLP buffers. I never merged because of a weird conflict a while ago (I guess it ended up being a lot easier to sort out...)